project/src/main/editor/creature/creature-editor-shadows.gd

```diff
@@ -0,0 +1,15 @@
+extends Node2D
+"""
+Manages shadows for the creature editor.
+"""
+
+export (PackedScene) var CreatureShadowScene: PackedScene
+
+func _ready() -> void:
+	# create shadows for all creatures in the scene
+	for creature_node in get_tree().get_nodes_in_group("creatures"):
+		var creature: Creature = creature_node
+		var creature_shadow: CreatureShadow = CreatureShadowScene.instance()
+		creature_shadow.shadow_scale = creature.scale * 0.4
```

Comment:
This '0.4' value could be extracted from the Creature so it's not hard-coded in all these different places. Maybe for refactoring day.